vm-mock/vm.js

```diff
@@ -104,32 +115,26 @@ export default function createMockedABI(memory, createImports, instantiate, bina
           }
         }
 
-        return webModule.__newArrayBuffer(v);
+        return getResult(v);
       },
 
-      // XXX: Untested ABI, directly migrated from as-tester
-
-      // TODO: Please move the ABIs to the previous section once you have confirmed
-      //        that they are functional.
-      // This action can be done on the fly.
-
       assembly_script_reset_storage() {
         resetLedger();
       },
 
       assembly_script_change_call_stack(callstackPtr) {
-        const callStackToString = webModule.__getArrayBuffer(callstackPtr);
+        const callStackToString = getString(callstackPtr);
         callStack = callStackToString;
       },
 
       assembly_script_generate_event(string) {
-        console.log('event', webModule.__getArrayBuffer(string));
+        console.log('event: ', getString(string));
       },
 
       assembly_script_set_data_for(aPtr, kPtr, vPtr) {
-        const a = webModule.__getArrayBuffer(aPtr);
-        const k = webModule.__getArrayBuffer(kPtr);
-        const v = webModule.__getArrayBuffer(vPtr);
+        const a = getString(aPtr);
+        const k = getKey(kPtr);
+        const v = getValue(vPtr);
```

Comment:
I did the same as line 99 and 100

Reply:
getValue is the same as before, for get key we can do whatever we want because it's the key of the hasmap, so I took what have been done for assembly_script_set_data, which is noted to be tested.